q2_types/_feature_table.py

```diff
@@ -97,6 +97,9 @@ def feature_table_to_pandas_dataframe(data_dir):
 plugin.register_semantic_type(RelativeFrequency)
 plugin.register_semantic_type(PresenceAbsence)
 
-plugin.register_type_to_data_layout(
-    FeatureTable[Frequency | RelativeFrequency | PresenceAbsence],
```

Comment:
See https://github.com/qiime2/qiime2/pull/104 for a better solution. I reverted the registration changes here, it'll work once https://github.com/qiime2/qiime2/pull/104 is merged.